.gitlab/deps_fetch/deps_fetch.yml

```diff
@@ -60,6 +60,7 @@ go_deps:
         prefix: "go_deps_modcache"
       paths:
         - modcache.tar.xz
+  timeout: 35m
```

Comment:
Could you explain what strategy did you use to set the timeout durations. If I'm correct, the first one is either `p100 x 3` or `p100 + 10mn` and for this one it's `p100 + 12mn` (or `p100 + 10mn rounded at the closest n x 5mn`)?
Thanks in advance

Reply:
The $+ 10$ is due to the time we have to wait to find a runner